Makefile

```
@@ -0,0 +1,61 @@
+	@echo "  make test       - Run tests"
+	@echo "  make lint       - Run linters"
+	@echo "  make format     - Format code"
+	@echo "  make verify     - Verify links"
+	@echo "  make build      - Build Docker image"
+	@echo "  make clean      - Clean build artifacts"
+
+install:
+	python3 -m pip install --upgrade pip
+	pip install -r requirements.txt
+
+test:
```

checkmake (0.2.2)

[warning] Target body for "test" exceeds allowed length of 5 (8).

(maxbodylength)